src/main/scala/go/Api.scala

```diff
@@ -0,0 +1,334 @@
+package strategygames.go
+
+import com.joansala.game.go.GoGame
+import com.joansala.game.go.GoBoard
+
+import cats.implicits._
+
+import strategygames.{ Pocket, Pockets }
+import strategygames.go.format.FEN
+import strategygames.go.Pos
+import strategygames.go.variant.Variant
+import scala.util.Try
+
+sealed abstract class GameResult extends Product with Serializable
+
+object GameResult {
+  final case class VariantEnd() extends GameResult
+  final case class Draw()       extends GameResult
+  final case class Ongoing()    extends GameResult
+
+  def resultFromInt(value: Int, ended: Boolean): GameResult =
+    if (value.abs == 1000 && ended) GameResult.VariantEnd()
+    else if (value == 0 && ended) GameResult.Draw()
+    else if (!ended) GameResult.Ongoing()
+    else sys.error(s"Unknown game result: ${value}")
+
+}
+
+object Api {
+
+  abstract class Position {
+    val variant: Variant
+
+    def makeMoves(movesList: List[Int]): Position
```

Comment:
might make sense to rename this to makeActions or something - Go doesnt have Moves so seems a little strange

Reply:
added todo for later.